tests/python/gaia-ui-tests/gaiatest/mixins/gaia.py

```diff
@@ -19,6 +19,11 @@ def __init__(self, **kwargs):
                          default=False,
                          help='restart target instance between tests')
 
+        group.add_option('--locale',
+                         default="undefined",
```

Comment:
I don't think we should put a default for language.current in there. I think the default of the pone is without anything set in language.current and it defaults to a language just fine (en-US).